app/assets/javascripts/govuk-component/govspeak-convert-html-pub-charts.js

```diff
@@ -1,8 +1,9 @@
 function govspeakBarcharts() {
-    $('.govuk-govspeak .js-barchart-table').each(function() {
+    $('.govuk-govspeak .js-barchart-table:not(.mc-chart):not(.js-barchart-table-init)').each(function() {
```

Comment:
This selector seems rather flaky. Isn't there a way to check for a `magnaCharta` instance instead? Or update the script to ensure is doesn't load twice.

Reply:
It is grim I agree, it's quite hard to deal with as the library seems rather legacy.

An alternative selector could be `.govuk-govspeak table.js-barchart-table:not(.js-barchart-table-init)` - I don't think there's another way without changing the library code - are you happier with that? I know it's a choice between two non ideal scenarios.

I'm not keen on changing the library itself as that involves picking up a project that hasn't been touched for 4 years and will easily escalate as a piece of work.

It might be more relevant to have this chat on https://github.com/alphagov/govuk_publishing_components/pull/751/files#diff-bd92d59aa72dc924833d655049e37ee0R17 as the code in this repo is intended to be very short lived whereas the code in govuk_publishing_components will have a longer life

Reply:
Good point about the not-so-long life of this repo. Happy to get it merged.